### PR TITLE
Change update available text and button url

### LINKED ISSUE #1776: Change the text in the "Update Available" dialog box
Please can you update the notification that alerts app users to updates so that it points to the page where you are able to see the breaking changes warning before they download it. 

So that the users are aware as soon as possible that the download will require a full reinstall of all mods etc. 

We are getting more and more users who will already have the app installed and will need to know about breaking changes in a more obvious way. 

When we move to a more robust migration system it will mitigate this. 

New text:

Update available
{OLD} -> {NEW}
Please note: In order to update to the latest version of the app you need to uninstall all your mods and the current version of the app. [Read more](http://link/).
[Later] [View Changelog]

### PR DESCRIPTION
**Pull request by insomnious**
Closes #1776

### CODE REVIEW

File: src/NexusMods.App.UI/Resources/Language.resx [hunk 1]
```diff
@@ -46,13 +46,13 @@
...
     <data name="Updater_UPDATE" xml:space="preserve">
-        <value>Update</value>
+        <value>View update</value>
     </data>
     <data name="Updater_UpdateAvailable" xml:space="preserve">
         <value>Update available</value>
     </data>
     <data name="Updater_MainMessage" xml:space="preserve">
-        <value>Keep your app updated to get the latest improvements.</value>
+        <value>Please note: In order to update to the latest version of the app you need to uninstall all your mods and the current version of the app.</value>
```
[the remark this answers is not in the document]
Comment (insomnious) [reply src/NexusMods.App.UI/Resources/Language.resx:55]: @captainsandypants ok to go ahead with this text change? and the instructions would be [this link](https://nexus-mods.github.io/NexusMods.App/users/Uninstall/), where shall we put that?

[the remark this answers is not in the document]
Comment (insomnious) [reply src/NexusMods.App.UI/Resources/Language.resx]: Done

**Review by Al12rs — APPROVED** [review]
Note: This removes the link to the changelog file on github (not release page, but actual changelog file).

If @captainsandypants is ok with changes we can merge

**Comment by captainsandypants** [comment]
> Note: This removes the link to the changelog file on github (not release page, but actual changelog file).
> 
> If @captainsandypants is ok with changes we can merge

Yeah all goof

**Review by captainsandypants — APPROVED** [review]

**Pull request merged**